_includes/faq/auto-generate-unique-ids_v1.1.html

```diff
@@ -0,0 +1,19 @@
+To auto-generate unique row IDs, use the [`UUID`](uuid.md) column with the `uuid_v4()` function cast to `UUID` as the default value:
+
+~~~ sql
+> CREATE TABLE new_test (id UUID PRIMARY KEY DEFAULT uuid_v4()::UUID, name STRING);
+~~~
+
+Alternatively, you can use the [`BYTES`](bytes.html) column with the `uuid_v4()` function as the default value instead:
+
+~~~ sql
+> CREATE TABLE test (id BYTES PRIMARY KEY DEFAULT uuid_v4(), name STRING);
+~~~
+
+In cases where you specifically need time-locality for unique row IDs, use the [`SERIAL`](serial.html) data type, which is an alias for [`INT`](int.html) with the `unique_rowid()` [function](functions-and-operators.html#id-generation-functions) as the [default value](default-value.html). On insert, the `unique_rowid()` function generates a default value from the timestamp and ID of the node executing the insert:
+
+~~~ sql
+> CREATE TABLE test (id SERIAL PRIMARY KEY, name STRING);
+~~~
+
+The `SERIAL` values are likely to be globally unique except in cases where a very large number of IDs (100,000+) are generated per node per second. Because BYTES and UUID values are 128-bit, much larger than INT values at 64-bit, there is virtually no chance of generating non-unique values. The distribution of IDs at the key-value level may also be a consideration. When using `BYTES` or 'UUID' with `uuid_v4()` as the default value, consecutively generated IDs will be spread across different key-value ranges (and therefore likely across different nodes), whereas when using `INT` with `unique_rowid()` as the default value, consecutively generated IDs may end up in the same key-value range. Hence in most cases, use `BYTES` or `UUID` data types to auto-generate unique row IDs.
```

Comment:
A few minor comments:
- Need to put backticks around `UUID`, `BYTES`, and `INT` in the second sentence. 
- Let's make "The distribution of IDs at the key-value layer" a new paragraph.
- In `When using `BYTES` or 'UUID' with `uuid_v4()` as the default value`, you need to change the single quotes around `UUID` to backticks to get the code-formatting.
- The last sentence, "Hence in most cases" feels kind of out-of-place. Let's just remove it. I think the explanations are sufficient.